OWNERS

```diff
@@ -1 +1,2 @@
 Allen-1212
```

Comment:
Cannot parse file: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go value of type repoowners.FullConfig.